### PR TITLE
chore: explicitly list published files

### PR DESCRIPTION
**Pull request by barmac**
> Explicit is better than implicit.

~Zen of Python

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,11 @@
   "name": "dmn-moddle",
   "version": "9.1.0",
   "description": "A moddle wrapper for DMN 1.3",
+  "files": [
```
Comment (nikku) [inline package.json:5]: We don't usually do this but rather rely on `.npmignore` to be complete. What is the benefit of this one?

Comment (barmac) [reply package.json:5]: You don't need to remember to `npmignore` new files as it was in the recent housekeeping PR.

Comment (nikku) [reply package.json:5]: Please put this up for discussion in our weekly. Happy to merge if we plan to adopt it on a broader scale.

Comment (barmac) [reply package.json:5]: We can do that, but note that we have already used that in some projects: https://github.com/bpmn-io/generator-diagram-js/blob/39b5594edd14a8d27d83d7a830df87ef07f927cd/package.json#L10, https://github.com/bpmn-io/bpmn-font/blob/b366fdef83c700898a664cb32d57d63618b36154/package.json#L26

Comment (barmac) [reply package.json:5]: Anyway, I'll talk more on package.json fields in the lightning talk.

Comment (nikku) [reply package.json:5]: Just to be clear: I'm very well aware what it does and use it for personal projects, too.

However, maintaining our zoo of tools consistency is important. If there is multiple ways to do things it makes a lot of sense to clearly communicate if we follow approach A or B in the future. Especially in places that are more critical than others (ignoring files from pushing, publishing assets, ...). 

There may be two side-projects where we're already using the `package.json` files configuration, great. Happens after creating 150 projects in seven years :wink:. If we do not clearly see a benefit in either, we should aim to use one of the approaches consistently.

A slot in our knowledge sharing round or a slack message to announce the _new cool way_ is the least we can do. So :+1: on the lightning talk. Maybe we can decide to adopt this way in the future, if you can argue for it.

Comment (nikku) [reply package.json:5]: We've decided to take this route. Thanks for driving the discussion :clap:

**Pull request merged**